cookbooks/fb_systemd/recipes/default.rb

```diff
@@ -128,6 +128,13 @@
   mode '0755'
 end
 
-link '/etc/systemd/system/default.target' do
-  to lazy { node['fb_systemd']['default_target'] }
+execute 'set default target' do
+  only_if do
+    current = Mixlib::ShellOut.new('systemctl get-default').run_command.
```

Comment:
Should we bail out here if this fails? i.e. `error!`

Reply:
I don't believe it can fail, but even if it does, the set-default will fail because it validates the target is valid, which wills till bail out. 

That said I have no problems adding `fail ... if ...`

Reply:
get-default goes over dbus (https://github.com/systemd/systemd/blob/master/src/systemctl/systemctl.c#L2132), so it could fail if dbus is hosed or if pid 1 is deadlocked